deploy/chart/templates/0000_50_olm_08-catalog-operator.deployment.yaml

```diff
@@ -18,6 +18,11 @@ spec:
         app: catalog-operator
     spec:
       serviceAccountName: olm-operator-serviceaccount
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 1001
```

Comment:
This one cannot be qualified to access restricted-v2 because of the `runAsUser`
That will requires access to anyuid SCC. However, if we test in k8s env that can run in a restricted ns and is qualified under the k8s rules to be restricted.